src/faebryk/library/has_schematic_hints.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of the faebryk project
```

Comment:
@IoannisP-ITENG I've got more hints to add to this, and imagine it'll grow with time.

What do you think of this pattern?